cmd/query/app/http_handler.go

```diff
@@ -428,8 +430,19 @@ func (aH *APIHandler) parseMicroseconds(w http.ResponseWriter, r *http.Request,
 	return time.Time{}, true
 }
 
-func (aH *APIHandler) parseGetTraceParameters(w http.ResponseWriter, r *http.Request) (spanstore.GetTraceParameters, bool) {
-	query := spanstore.GetTraceParameters{}
+func (aH *APIHandler) parseBool(w http.ResponseWriter, r *http.Request, boolKey string) (value bool, isValid bool) {
+	if boolString := r.FormValue(boolKey); boolString != "" {
+		b, err := parseBool(r, boolKey)
+		if aH.handleError(w, err, http.StatusBadRequest) {
+			return false, false
+		}
+		return b, true
+	}
+	return false, true
+}
+
+func (aH *APIHandler) parseGetTraceParameters(w http.ResponseWriter, r *http.Request) (querysvc.GetTraceParameters, bool) {
```

Comment:
Side note: this function is poorly designed. A better design is `query_parser.parseTraceQueryParams`, which simply returns errors. The `aH.handleError` should really be used only rarely, directly in the handler function.

Reply:
do we want to address this in a future PR?

Reply:
it's one of these nice to have clean-ups, I'd rather have it as good-first-issue for someone other than you spending time on it.